qcodes/tests/test_group_parameter.py

```diff
@@ -66,7 +77,6 @@ def test_sanity():
     assert dummy.a() == 3
     assert dummy.b() == 10
 
-
 def test_raise_on_get_set_cmd():
```

Comment:
please, don't remove the space - per pep8 there should be two empty lines between functions.